core/src/components/input/input.tsx

```diff
@@ -305,6 +305,11 @@ export class Input implements ComponentInterface {
   /**
    * Sets focus on the native `input` in `ion-input`. Use this method instead of the global
    * `input.focus()`.
+   *
+   * Angular Note:
+   * This method should not be called from Angular's `ngAfterViewInit()` lifecycle hook
+   * because the underlying `<input>` element may not be loaded at that time.
+   * Use Ionic's `ionViewDidEnter()` lifecycle method instead.
```

Comment:
Can we update this note to say that `ionViewDidEnter` should be used to focus an element when a page enters? We try to avoid framework-specific comments in the Web Component properties since all developers will see this documentation.

Maybe something like:

```
Developers who wish to focus an input when a page enters should call `setFocus` in the `ionViewDidEnter` lifecycle method.
```

Reply:
That sounds good to me, I've updated the PR!